src/Imdb/Title.php

```diff
@@ -1621,6 +1632,59 @@ public function cast($short = false)
         if (empty($page)) {
             return array(); // no such page
         }
+        if ($short && $this->detect_new_version_imdb()) {
+            $xpath = $this->getXpathPage("Title");
+            $nodes = $xpath->query("//section[@data-testid='title-cast']/div[2]/div[@data-testid='title-cast-item']");
+            foreach ($nodes as $i => $node) {
+                $dir = array(
+                    'imdb' => null,
+                    'name' => null,
+                    'name_alias' => null,
+                    'credited' => true,
+                    'role' => null,
+                    'role_episodes' => null,
+                    'role_start_year' => null,
+                    'role_end_year' => null,
+                    'role_other' => array(),
+                    'thumb' => null,
+                    'photo' => null
+                  );
+                $get_name_and_id = $xpath->query(".//a[@data-testid='title-cast-item__actor']", $node)->item(0);
+                $dir['imdb'] = preg_replace('/\/?name\/nm(\d+)[\/\?]+.*?$/is', '$1', $get_name_and_id->getAttribute("href"));
+                $dir["name"] = trim($get_name_and_id->nodeValue);
+                if (empty($dir['name'])) {
+                    continue;
+                }
+                $get_role = $xpath->query(".//a[@data-testid='cast-item-characters-link']/span[1]", $node);
+                if($get_role != null){
+                    $dir["role"] = $get_role->item(0)->nodeValue;
+                }
+
+                $get_img = $xpath->query(".//img[@class='ipc-image']", $node);
+                if($get_img != null && $get_img->item(0)->getAttribute("src") != null){
```

Comment:
This doesn't work, I get `Error : Call to a member function getAttribute() on null` when running the test case for short cast

Reply:
$get_img can not be false if query is provided with a valid node and xpath, you get an empty DOMNodeList 

https://www.php.net/manual/en/domxpath.query.php
`Returns a DOMNodeList containing all nodes matching the given XPath expression. Any expression which does not return nodes will return an empty DOMNodeList.`